functions/src/modules/email.functions.ts

```diff
@@ -0,0 +1,100 @@
+import * as functions from 'firebase-functions';
+import * as  admin from 'firebase-admin';
+
+const bucket = admin.storage().bucket();
+const db = admin.database();
+import * as helpers from './../helpers';
+
+const path = require('path');
+const os = require('os');
+const fs = require('fs');
+
+
+let emailTemplates = {};
```

Comment:
Repeating: this variable is never reassigned or pushed to. What is the purpose of it?

Reply:
It's used for creating a caching index ... in `sendNotificationEmail` 
```js
if (Object.keys(emailTemplates).indexOf(templateName) > -1)
        id = emailTemplates[templateName].id;
```

Reply:
Yes I see, but you're reading from this index here, and not writing anywhere. That's what confused me. Can you clarify this point?

Reply:
Right, added a new line to push found templates to the object.